abjad/tools/schemetools/Scheme.py

```diff
@@ -120,21 +120,23 @@ class Scheme(AbjadValueObject):
     ### INITIALIZER ###
 
     def __init__(self, *args, **kwargs):
-        if 1 == len(args):
+        if len(args) == 1:
             if isinstance(args[0], type(self)):
-                args = args[0]._value
+                value = args[0]._value
             else:
-                args = args[0]
-        quoting = kwargs.get('quoting')
+                value = args[0]
+        else:
+            value = args
         force_quotes = bool(kwargs.get('force_quotes'))
-        verbatim = kwargs.get('verbatim')
-        assert isinstance(quoting, (str, type(None)))
+        verbatim = bool(kwargs.get('verbatim'))
+        quoting = kwargs.get('quoting')
+        assert isinstance(quoting, str) or quoting is None
```

Comment:
Why not:

```
        if quoting is not None:
            assert isinstance(quoting, basestring)
            assert all(c in r"',@`#" for c in quoting)
```

I'd also prefer `character` to `c`. It's a lot easier to grep in our codebase, which is literally littered with the character `c` due to the amount of LilyPond code.